.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -13,7 +13,7 @@ jobs:
   steps:
     - task: CondaEnvironment@1
       inputs:
-        packageSpecs: 'python=3.6 conda-build>=3.18 conda>4.7.12 conda-forge::conda-forge-ci-setup=3 networkx=2.4 conda-forge-pinning' # Optional
+        packageSpecs: 'python=3.9 conda-build>=3.18 conda>4.7.12 conda-forge::conda-forge-ci-setup=3 networkx=2.4 conda-forge-pinning' # Optional
```

Comment:
```suggestion
        packageSpecs: 'python=3.8 conda-build>=3.18 conda>4.7.12 conda-forge::conda-forge-ci-setup=3 networkx=2.4 conda-forge-pinning' # Optional
```

Reply:
Sure we can do that. Should we be doing the same thing in conda-smithy ( https://github.com/conda-forge/conda-smithy/pull/1504 )?

Reply:
IDK why 3.9 isn't working. We should file an issue on the libarchive feedstock maybe. If we can get 3.8 to work, then we can move to that for the time being.

Reply:
Sounds good. Just didn't know if I was missing something :)